### DISCUSSION TITLE
Does the server enforce website icon settings?

### DISCUSSION THREAD

**Discussion by sfkpmr** [dani-garcia/vaultwarden#2503] [Q&A] [answered]
I've disabled website icons for my server, but found that the same setting exists in the browser extensions and apps too. 

Do you need to disable this everywhere, or is it enough to disable it in the server?

**Comment by BlackDex** [comment, marked as answer]
If you disable ot on the clients then they won't even initiate a call to download icons at all. If you disable it on the server side, the server will not download icons, but does still receive calls to them, but just doesn't answer with a valid icon.

**Reply by sfkpmr** [reply]
Ok, so it all runs through the server? Thanks

**Reply by BlackDex** [reply]
Yes all calls go through the server. Unless you configure it otherwise.
